src/Controller/GUIController.py

```diff
@@ -2,10 +2,11 @@
 from PyQt5 import QtCore, QtWidgets, QtGui
 from PyQt5.QtWidgets import QMessageBox
 
-from src.View.mainpage.MainPage import UIMainWindow
+from src.Model.PatientDictContainer import PatientDictContainer
```

Comment:
we didn't really talk about sorting imports but, it's considered best practice.
See https://pypi.org/project/isort/ and https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement#20763446